Services/StaticJobProvider.php

```diff
@@ -109,4 +113,44 @@ protected function throwRegexMatchException(string $configurationHostname, strin
 
         throw new \RuntimeException($exceptionMessage);
     }
+
+    /**
+     * Validate hostname
+     *
+     * @param mixed $hostname
+     *
+     * @return void
+     * @throws \Exception
+     */
+    private function validateHostname($hostname)
+    {
+        if (!is_string($hostname)) {
+            throw new \Exception(
```

Comment:
imo it's a bad idea to throw generic exceptions

Reply:
@oleg-andreyev done.